### DISCUSSION TITLE
pg_hba.conf里面trust全部改为md5[Ideas]

### DISCUSSION THREAD

**Discussion by Aken-Gan** [apache/cloudberry#356] [Ideas / Feature Requests]
### Description

目前cbdb的master和segment节点的pg_hba.conf文件使用trust认证，不符合部分客户安全审计要求，客户提出必须全部使用md5方式，目前直接手工修改配置，会到最后gpstop、gpstart、gpconfig等操作异常，因此需要内核层面支持，并将pg_hba.conf的默认配置改为md5。

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by tuhaihe** [comment]
Hi @Aken-Gan Thank you for your idea. Could you please rewrite it in English so that the global community members can understand?
